docs/resources/cluster_v2.md

```diff
@@ -475,6 +475,7 @@ The following attributes are exported:
 * `unhealthy_node_timeout_seconds` - (Optional) Seconds an unhealthy node has to become active before it is replaced (int)
 * `max_unhealthy` - (Optional) Max unhealthy nodes for automated replacement to be allowed (string)
 * `unhealthy_range` - (Optional) Range of unhealthy nodes for automated replacement to be allowed (string)
+* `machine_labels` - (Optional) Machine pool labels. Node labels (list)
```

Comment:
I think this should be changed to `(Optional) Labels for a machine pool deployment (list)` to be more clear about what this is.

Reply:
I have updated the documentation to document both `labels` and `machine_labels`. I hope this is clearer now. Happy to revise again. I'm having trouble coming up with something that is clear :)